pkg/cmd/drt/scripts/tpcc_run.sh

```diff
@@ -34,7 +36,7 @@ while true; do
       --tolerate-errors \
       --password tpcc \
       --families \
-        "${PGURLS_ARR[@]}" | tee $LOG
+      $PGURLS | tee $LOG
```

Comment:
`--warehouses 3000, --active-warehouses 1500` value of this flags need to be changed.

Also we need to think about values of following flags to get the cpu around 60%.
```
-concurrency 128
--max-rate 7000 
```